### PR TITLE
Prevent NPE on delete local module cache

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
Prevent NPE on delete local module cache in HeliumBundleFactory

```
Caused by: java.lang.NullPointerException
	at org.apache.zeppelin.helium.HeliumBundleFactory.deleteYarnCache(HeliumBundleFactory.java:465)
	at org.apache.zeppelin.helium.HeliumBundleFactory.copyFrameworkModulesToInstallPath(HeliumBundleFactory.java:487)
	at org.apache.zeppelin.helium.HeliumBundleFactory.buildPackage(HeliumBundleFactory.java:403)
	at org.apache.zeppelin.helium.Helium.enable(Helium.java:314)
	at org.apache.zeppelin.rest.HeliumRestApi.enablePackage(HeliumRestApi.java:193)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:180)
	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96)
	... 50 more
```

### What type of PR is it?
Bug Fix

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/S3NotebookRepo.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ public S3NotebookRepo(ZeppelinConfiguration conf) throws IOException {
 
     if (StringUtils.isNotBlank(keyRegion)) {
       cryptoConf = new CryptoConfiguration();
-      cryptoConf.setAwsKmsRegion(Region.getRegion(Regions.fromName(keyRegion)));
+      //cryptoConf.setAwsKmsRegion(Region.getRegion(Regions.fromName(keyRegion)));
```
Comment (1ambda) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/S3NotebookRepo.java]: Is this line related with yarn caches?

Comment (Leemoonsoo) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/S3NotebookRepo.java]: This is unnecessary change. i rewrote the commit.

**Comment by 1ambda** [comment]
LGTM

**Comment by Leemoonsoo** [comment]
Thanks @1ambda for review. Merge to master if no further discussions.
